workbench-core/authorization/src/dynamicAuthorization/dynamicAuthorizationService.ts

```diff
@@ -96,12 +103,18 @@ export class DynamicAuthorizationService {
    * Create an authorization group
    * @param createGroupRequest - {@link CreateGroupRequest}
    *
-   * @returns - {@link CreateGroupResponse}
+   * @returns a {@link CreateGroupResponse}
    *
-   * @throws - {@link GroupAlreadyExistsError} Can not create a group that already exists
+   * @throws {@link GroupAlreadyExistsError} - Can not create a group that already exists
+   * @throws {@link PluginConfigurationError} - plugin has a configuration error
+   * @throws {@link TooManyRequestsError} - too many requests error
    */
   public async createGroup(createGroupRequest: CreateGroupRequest): Promise<CreateGroupResponse> {
-    throw new Error('Not implemented');
+    await this._groupManagementPlugin.setGroupStatus({
```

Comment:
If `createGroup` fails, do we need to do any status changes?

Reply:
fair point, I'll reorder the calls!